docs/mp/integrations/jedis.adoc

```diff
@@ -16,17 +16,28 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-= CDI extension for Jedis
-:description: Helidon CDI extension for Jedis
-:keywords: helidon, java, microservices, microprofile, extensions, cdi, jedis, redis
-:feature-name: Jedis Support
+= Jedis
+:description: Helidon Jedis integration
+:keywords: jedis, redis
+:feature-name: Jedis
 :rootdir: {docdir}/../..
 
 include::{rootdir}/includes/mp.adoc[]
 
+== ToC
+
+- <<Overview, Overview>>
+- <<maven-coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<Configuration, Configuration>>
+- <<References, References>>
+
+== Overview
+
+link:https://github.com/xetorthio/jedis[Jedis] is a Java client for link:https://github.com/redis/redis[Redis].
```

Comment:
The Jedis Github link here does not match the one at the end of this document.

Reply:
Fixed